spec/controllers/api/v2/application_controller_spec.rb

```diff
@@ -2,10 +2,13 @@
 
 describe API::V2::ApplicationController, type: :controller do
   describe '#authenticate' do
+    let(:secret) {
```

Comment:
When putting a block on multiple lines it's Ruby convention to use `do` `end`, this isn't too long though so I'd just put it on one line.